src/types.ts

```diff
@@ -93,10 +93,22 @@ const Policy = t.intersection([
   })
 ]);
 
+const IdentificationFlag = t.intersection([
+  t.type({
+    type: t.union([t.literal("STRING"), t.literal("NUMBER")]),
```

Comment:
Oh just noticed this,  we can use this to change the keyboard in the input id section